src/layout/dom.ts

```diff
@@ -118,10 +119,27 @@ export function update(node: Node, data: NodeInfo, source: Source): void {
     }
 }
 
+function position(parent: NodeValue, child: NodeValue): number {
+    let tag = child.data.tag;
+    // Find relative position of the element to generate :nth-of-type selector
+    // We restrict relative positioning to handful of tags for now.
+    if (parent && (tag === "DIV" || tag === "TR" || tag === "P" || tag === "LI")) {
```

Comment:
>TR [](start = 45, length = 2)

What about TD?

Reply:
Swaathee did some initial analysis to restrict the tags to select few to strike a balance between being unique enough and being too specific. She will cover this in her doc. I am assuming we will go through iterations here but going with the list she shared with me based on her research so far. So won't fix for now but definitely open to making changes after Swaathee's design review.